answers.txt

```diff
@@ -1,4 +1,10 @@
-1)  What is a gitignore file and what should it include for a Java project? 
-2)  What does fast-forward in git mean? 
-3)  Why should you have different branches? 
-4)  In a project how many remote Git repositories do you usually have?
\ No newline at end of file
+
+Answers : 
+
+Ans 1) gitignore file tells git which files or folders should be ignored in project when you commit the project.
+
+Ans 2) In fast-forward merge, git simply moves the source branch pointer to the target branch pointer without creating an extra merge commit.
+
+Ans 3) By having different branches in git the developers can branch out from the original code and can work on their part individually or also isolate their work from others. Plus it also help Git to merge versions easily when ever needed.
+
+Ans 4) We can have has many remote git repositories as we like but there can only be one remote named origin. It is the default remote created by Git, when you clone a current repository you can set some branches to look over other branches from second remote rather than origin.
```

Comment:
```suggestion
Ans 4) We can have as many remote git repositories as we like but there can only be one remote named origin. It is the default remote created by Git, when you clone a current repository you can set some branches to look over other branches from second remote rather than origin.
```

Reply:
Added suggestion for a typo fix.